helm/prometheus-rules/templates/alerting-rules/app.rules.yml

```diff
@@ -107,12 +107,26 @@ spec:
         sig: none
         team: rocket
         topic: releng
-    ## This rule is only available on the management cluster prometheus becasue the source is the app operator
+    - alert: ManagementClusterAppPendingUpdate
+      annotations:
+        description: 'Current version of {{`App {{ $labels.name }} is {{ $labels.deployed_version }} but it should be {{ $labels.version }}.`}}'
+      expr: app_operator_app_info{catalog=~"control-plane-.*", deployed_version!="", status="deployed", version_mismatch="true"}
+      for: 40m
+      labels:
+        area: managedservices
+        cancel_if_apiserver_down: "true"
+        cancel_if_cluster_status_creating: "true"
+        cancel_if_cluster_status_deleting: "true"
+        cancel_if_cluster_status_updating: "true"
+        severity: notify
+        team: batman
+        topic: releng
+    ## Workload cluster app rules only available on the management cluster Prometheus because the source is app-exporter
     - alert: WorkloadClusterAppFailed
       annotations:
         description: '{{`Workload Cluster App {{ $labels.name }}, version {{ $labels.version }} is {{if $labels.status }} in {{ $labels.status }} state. {{else}} not installed. {{end}}`}}'
         opsrecipe: app-failed/
-      expr: label_replace(app_operator_app_info{status!~"(?i:(deployed|cordoned))", catalog!~"control-plane-.*", name!="cluster-autoscaler"}, "cluster_id", "$1", "namespace", "(([^g]|g[^i]|gi[^a]|gia[^n]|gian[^t]|giant[^s]|giants[^w]|giantsw[^a]|giantswa[^r]|giantswar[^m])*)") == 1
+      expr: label_replace(app_operator_app_info{status!~"(?i:(deployed|cordoned))", catalog!~"control-plane-.*"}, "cluster_id", "$1", "namespace", "(([^g]|g[^i]|gi[^a]|gia[^n]|gian[^t]|giant[^s]|giants[^w]|giantsw[^a]|giantswa[^r]|giantswar[^m])*)") == 1
```

Comment:
We need the replace magic to set the cluster ID. Current alerts show as MC apps

![Screen Shot 2021-06-29 at 6 16 46 PM](https://user-images.githubusercontent.com/311527/123833400-95baa380-d906-11eb-88cb-ebbaf3852ff6.png)

Reply:
To me, the only difference looks like coming from the deletion of `name!="cluster-autoscaler"}` is this right?

Reply:
Yes, sorry I added the comment in the  wrong place.